### PR TITLE
Hamburger menu

### PR DESCRIPTION
**Pull request by oskber**
Added functionality for hamburger menu. Thoughts?

image

image

### CODE REVIEW

File: kino-nextjs/app/api/movies/[id]/movies.ts
Comment (MickeMarten) [inline kino-nextjs/app/api/movies/[id]/movies.ts:1]: I think this is an older file that is deleted in the new version?

**Review by MickeMarten — APPROVED** [review]
Looks awesome. Nice job with the types.

File: kino-nextjs/app/api/movies/[id]/movies.ts
Comment (oskber) [reply kino-nextjs/app/api/movies/[id]/movies.ts:1]: It is possible, should I delete it?

Comment (ANschoolacc) [reply kino-nextjs/app/api/movies/[id]/movies.ts:1]: Yes. This file does not exist in the main branch and has no purpose.

Comment (ANschoolacc) [reply kino-nextjs/app/api/movies/[id]/movies.ts:1]: The folder /api/movies that is

**Review by ANschoolacc — APPROVED** [review]
Nice job with the hamburger menu, it looks really good! 
But remove the api/movies folder before merging as it is outdated.

**Review by maxlin94 — APPROVED** [review]
Looks good!

**Comment by ConnyPettersson** [comment]
Hamburger menu looks good!

**Pull request merged**